pages/package-manager.qmd

```diff
@@ -1,19 +1,21 @@
 ## Posit Package Manager
-There are two ways for Posit Package Manager to pick up updates from R packages on Github (not applicable to CRAN packages): `tags` or `commits`. Commits mode is based on all commits made to a Git branch, while tags mode is based on Git tags in the package repository. 
+PHS have an instance of [Posit Package Manager](https://posit.co/products/enterprise/package-manager/), - [https://ppm.publichealthscotland.org/client/]() this provides a seamless interface for users when installing packages either from CRAN, or packages hosted elsewhere - this is how the PHS packages can be installed easily even though they are only hosted on GitHub. 
```

Comment:
Set link to open in a new tab `{target="_blank"}`

Reply:
I've added `{target="_blank"}` throughout, I thought it would be good to just set that as default in the YAML but apparently, that's not possible yet [https://github.com/quarto-dev/quarto-cli/issues/10730]() I left the link with empty `()` as that's valid markdown and so should work but let me know if doesn't after building!

Reply:
@Moohan - seems to link back to the same page you are on unfortunately, this is the link it produces for me http://localhost:6317/pages/package-manager.html

Reply:
> I've added {target="_blank"} throughout, I thought it would be good to just set that as default in the YAML but apparently, that's not possible yet [https://github.com/quarto-dev/quarto-cli/issues/10730](https://github.com/Public-Health-Scotland/package-dev-guidance-site/pull/24).

This would be great if possible!